custom_components/places/coordinator.py

```
@@ -0,0 +1,511 @@
+
+    def set_attr(self, attr: str, value: object | None = None) -> None:
+        """Store a runtime attribute."""
+        self._attributes.set(attr, value)
+
+    def clear_attr(self, attr: str) -> None:
+        """Remove a runtime attribute."""
+        self._attributes.clear(attr)
+        if attr == ATTR_NATIVE_VALUE:
+            self._native_value = None
+
+    def set_native_value(self, value: Any) -> None:
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)